packages/text-input/src/index.tsx

```diff
@@ -35,6 +35,10 @@ const TextInput: React.FC<TextInputProps> = ({
 export const TextInputComponents = S
 export {TextInput, TextInputProps, TextInputAppearanceType}
 export {getTextInputStyle} from './styles'
+export {
```

Comment:
In future I would like to create `@smashing/ui` package which will export every component exports so we need to keep them clean.

```tsx
export const TextInputElements = {
  Container: StyledTextContainer,
  Input,
}
```

Reply:
done